### DISCUSSION TITLE
Size of segemented buttons in tabview

### DISCUSSION THREAD

**Discussion by zSynctic** [TomSchimansky/CustomTkinter#1751] [Q&A] [answered]
How can I change the size of the segemented buttons in tabview

**Comment by Akascape** [comment]


**Reply by zSynctic** [reply]
what is the first segmented_button part

**Reply by Akascape** [reply]
@zSynctic That is the name of your segmented button.

**Reply by zSynctic** [reply]
My question is about the buttons that are like segmented buttons in tabview so is there a way to do that?

**Reply by Akascape** [reply, marked as answer]
@zSynctic 
Same segmented button is used in tabview:
```python
tabview._segmented_button._buttons_dict["button_name"].configure(width=200)
```

**Reply by zSynctic** [reply]
Oh ok thanks

**Reply by zSynctic** [reply]
I get an error: Exception has occurred: AttributeError
'CTkSegmentedButton' object has no attribute 'option_type' when doing that

**Reply by Akascape** [reply]
@zSynctic sorry, remove that option_type

**Reply by zSynctic** [reply]
np, thanks
